docs/reference/urllib3.response.rst

```diff
@@ -4,10 +4,20 @@ Response and Decoders
 Response
 --------
 
+.. autoclass:: urllib3.response.BaseHTTPResponse
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
 .. autoclass:: urllib3.response.HTTPResponse
     :members:
     :undoc-members:
     :show-inheritance:
+    :inherited-members:
```

Comment:
If possible we want to keep inherited members private except for common ones like `read()`. Is there a way we can opt-in to the members we want?

Reply:
Yes, we just have a warning for `json`. I've limited it to `json`